modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -513,6 +513,100 @@ public void deleteObjectRelationships(
 		}
 	}
 
+	@Override
+	public void disableEdge(long objectDefinitionId2) throws PortalException {
+		ObjectDefinition objectDefinition2 =
+			_objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId2);
+
+		for (ObjectRelationship objectRelationship :
+				getObjectRelationshipsByObjectDefinitionId2(
+					objectDefinitionId2)) {
+
+			if (!objectRelationship.isEdge()) {
+				continue;
+			}
+
+			ObjectDefinition objectDefinition1 =
+				_objectDefinitionPersistence.findByPrimaryKey(
+					objectRelationship.getObjectDefinitionId1());
+
+			if (objectDefinition1.getRootObjectDefinitionId() !=
+					objectDefinition2.getRootObjectDefinitionId()) {
+
+				objectRelationship.setEdge(true);
```

Comment:
@G-Sa shouldn't it be `objectRelationship.setEdge(false);`?

Reply:
![image](https://github.com/liferay-objects/liferay-portal/assets/60936590/03663b6a-f91c-4ce9-985a-6ddcb361f650)